docs/zello-sdk-documentation/index-all.html

```diff
@@ -582,6 +591,11 @@ <h2 class="title">S</h2>
 <div class="block">
      Sets the current <code>AudioMode</code> to the <a href="com/zello/sdk/AudioMode.html#WEARABLE"><code>AudioMode.WEARABLE</code></a> mode using the passed in wearable index.</div>
 </dd>
+<dt><span class="memberNameLink"><a href="com/zello/sdk/Zello.html#showMicrophonePermissionDialog--">showMicrophonePermissionDialog()</a></span> - Method in class com.zello.sdk.<a href="com/zello/sdk/Zello.html" title="class in com.zello.sdk">Zello</a></dt>
+<dd>
+<div class="block">Opens a popup dialog that tells the user that they cannot send voice messages until the
```

Comment:
We might add theme support for this dialog in the future, but for now it will show as the default theme for the ZelloWork app.